.github/ISSUE_TEMPLATE/add-instance.md

```diff
@@ -11,6 +11,10 @@ assignees: ''
 * [ ] It is my instance.
 * [ ] I give the right to searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
 
+Pick one:
+* [ ] I have install [filtron](https://github.com/asciimoo/filtron/), [antibot-proxy](https://github.com/unixfox/antibot-proxy) or another bot protection.
```

Comment:
I'm not a native English speaker but I think `Ì have install` is grammatically incorrect and should be instead `Ì have installed`.